client/src/App.jsx

```diff
@@ -1,21 +1,56 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import { MuiThemeProvider } from "@material-ui/core";
 import { BrowserRouter, Route, Redirect } from "react-router-dom";
+import axios from "axios";
+
+import UserContext from "./context/UserContext";
 import theme from "themes/theme";
 import SignUp from "components/SignUp";
 import Login from "components/Login";
 
+import Home from "pages/Home";
+
 import "./App.css";
 
 function App() {
+  const [user, setUser] = useState(null);
+
+  const logout = () => {
+    setUser(null);
```

Comment:
I guess this will depend on you and @gersonesp , however we will also want to remove the JWT token, and that's going to depend on how we store it - we can discuss further in standup, who is working on that and how we can do it

Reply:
I added code to remove the token from localStorage. I'm calling the data "token" for now (we can agree on a name for that token later)